src/pattern-library/components/patterns/UsingComponentsPage.tsx

```diff
@@ -1,331 +1,320 @@
           </p>
-        </Library.Pattern>
-      </Library.Section>
+          <Library.Example title="children">
+            <Library.Info>
+              <Library.InfoItem label="type">
+                <code>preact.ComponentChildren</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="classes">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                CSS class(es) that will be appended to the CSS classes applied
+                to the {"component's"} outermost or most significant element.
+                This can be used independently or in conjunction with the
+                Styling API props.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>string</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="elementRef">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                A <code>Ref</code> applied to the {"component's"} outermost or
+                most significant element.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>{'preact.Ref<HTMLElement | undefined>'}</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="...htmlAttributes">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Presentational components accept HTML attributes applicable to
+                the element type of the {"component's"} outermost or most
+                significant element.
+              </Library.InfoItem>
+              <Library.InfoItem label="example">
+                <Library.Code
+                  content={`<Link id="my-own-id" href="https://www.example.com">A Link with attributes</Link>`}
+                  size="sm"
+                />
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>
+                  {'preact.JSX.HTMLAttributes<T extends HTMLElement>'}
+                </code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+        </Library.Section>
 
-      <Library.Section
-        title="Composite components"
-        intro={
-          <>
-            <p>
-              <strong>Composite components</strong> compose and style
-              presentational components in a particular way to satisfy a use
-              case and sometimes support more nuanced or advanced behavior.
-            </p>
-            <p>
-              Example: <code>ScrollBox</code>, <code>Panel</code>.
-            </p>
-          </>
-        }
-      >
-        <Library.Pattern title="Props API">
+        <Library.Section
+          id="presentational-components-styling-api"
```

Comment:
Added a few IDs by hand in these pages so I could cross link. Hope to automate these soon.